src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/TelemetryReporterTests.cs

```diff
@@ -0,0 +1,280 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Razor.Telemetry;
+using Microsoft.VisualStudio.Editor.Razor.Test.Shared;
+using Microsoft.VisualStudio.Telemetry;
+using Newtonsoft.Json.Linq;
```

Comment:
Scanning through, I didn't see how the tests use Newtonsoft.Json. Did I miss something?

Reply:
It's not, probably accidentally added by usings on paste. Removed